scripts/zones/Bibiki_Bay/mobs/Shens_Filtrate.lua

```diff
@@ -0,0 +1,17 @@
+-----------------------------------
+-- Area: Bibiki Bay
+--  Mob: Shen's Filtrate - Shen Elemental
+-----------------------------------
+local ID = require("scripts/zones/Bibiki_Bay/IDs")
+-----------------------------------
+local entity = {}
+
+entity.onMobSpawn = function(mob)
+    mob:addMod(xi.mod.REGEN, 120)
```

Comment:
Is this from wings? Couldn't find any source of these guys have regen.

Reply:
This is from my own research with a capture that Zach did for me on the capture discord.